.eslintignore

```diff
@@ -0,0 +1,2 @@
+public
```

Comment:
`public` contains files served to the browser. I am not sure if we can switch them to ES6 (e.g. `const` instead of `var`), therefore I am leaving them out of this PR.